Client/Frontend/Browser/BrowserToolbar.swift

```diff
@@ -36,102 +30,72 @@ class BrowserToolbar: UIView, UITextFieldDelegate, BrowserLocationViewDelegate {
 
     override init(frame: CGRect) {
         super.init(frame: frame)
-        viewDidInit()
+        initViews()
```

Comment:
We should stop initializing views in `init()` methods and instead do this at `viewDidLoad` it is a bad habit against the proper `UIViewController` lifecycle.

In this case it also means you can just get rid of all the `init()` methods and rename `initViews` to `viewDidLoad`, which makes the code a lot simpler.